Library/Homebrew/cleanup.rb

```diff
@@ -23,10 +24,27 @@ def self.disk_cleanup_size
       @disk_cleanup_size
     end
 
+    def self.add_unremovable_keg(keg)
+      @unremovable_kegs << keg
+    end
+
+    def self.unremovable_kegs
+      @unremovable_kegs
+    end
```

Comment:
It's used externally from the Cleanup command. I wrote it this way for consistency with the two `disk_cleanup_size` methods rather than a knowledge of idiomatic ruby (I'm a ruby n00b coming from a mostly C# background). I could reasonably drop the `add_unremovable_keg` method, but I think the accessor needs to remain.

Is the following more idiomatic in ruby?

```ruby
def self.unremovable_kegs
  @unremovable_kegs ||= []
end

def cleanup_keg(keg)
  # ...
  unremovable_kegs << keg
```

or:

```ruby
@unremovable_kegs = []
attr_accessor :unremovable_kegs

def cleanup_keg(keg)
  # ...
  unremovable_kegs << keg
```